src/services/pathCompletions.ts

```diff
@@ -37,7 +37,6 @@ namespace ts.Completions.PathCompletions {
             }
         }
         else {
-            // Check for node modules

```

Comment:
This comment's not really accurate any more, the function name is more accurate.